dev-haskell/hoogle/hoogle-5.0.18.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+
+COMMIT="4bd6688857874fd41febb998a6420daa4b0b695f"
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Haskell API Search"
+HOMEPAGE="https://hoogle.haskell.org/"
+SRC_URI=" https://github.com/ndmitchell/hoogle/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+PATCHES=( "${FILESDIR}/${PN}-5.0.18-disable-network-tests.patch" )
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-haskell/aeson:=[profile?]
+	dev-haskell/blaze-html:=[profile?]
+	dev-haskell/blaze-markup:=[profile?]
+	dev-haskell/cmdargs:=[profile?]
+	>=dev-haskell/conduit-1.3.0:=[profile?]
+	>=dev-haskell/conduit-extra-1.2.3.2:=[profile?]
+	dev-haskell/connection:=[profile?]
+	>=dev-haskell/extra-1.6.6:=[profile?]
+	>=dev-haskell/foundation-0.0.13:=[profile?]
+	dev-haskell/hashable:=[profile?]
+	>=dev-haskell/haskell-src-exts-1.22:=[profile?] <dev-haskell/haskell-src-exts-1.24:=[profile?]
+	>=dev-haskell/http-conduit-2.3:=[profile?]
+	dev-haskell/http-types:=[profile?]
+	dev-haskell/js-flot:=[profile?]
+	dev-haskell/js-jquery:=[profile?]
+	dev-haskell/mmap:=[profile?]
+	dev-haskell/old-locale:=[profile?]
+	dev-haskell/process-extras:=[profile?]
+	dev-haskell/quickcheck:2=[profile?]
+	dev-haskell/resourcet:=[profile?]
+	>=dev-haskell/semigroups-0.18:=[profile?]
+	dev-haskell/storable-tuple:=[profile?]
+	dev-haskell/tar:=[profile?]
+	dev-haskell/text:=[profile?]
+	dev-haskell/uniplate:=[profile?]
+	dev-haskell/utf8-string:=[profile?]
+	dev-haskell/vector:=[profile?]
+	dev-haskell/wai:=[profile?]
+	dev-haskell/wai-logger:=[profile?]
+	dev-haskell/warp:=[profile?]
+	dev-haskell/warp-tls:=[profile?]
+	dev-haskell/zlib:=[profile?]
+	>=dev-lang/ghc-7.10.1:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-1.22.2.0
+"
+
+S="${WORKDIR}/hoogle-${COMMIT}"
+
+src_test() {
+	export LD_LIBRARY_PATH=${S}/dist/build
```

Comment:
This overrides `LD_LIBRARY_PATH`. Ideally it should append to existing one if exists. For example `agda` uses `export LD_LIBRARY_PATH="${S}/dist/build${LD_LIBRARY_PATH+:}${LD_LIBRARY_PATH}"`